### DISCUSSION TITLE
Welcome to simple-llama-finetuner Discussions!

### DISCUSSION THREAD

**Discussion by lxe** [lxe/simple-llama-finetuner#31] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by TiltonLAW** [comment]
Oh hi. I am a lawyer looking to create an AI able to assist with research and writing tasks.  Currently, CoCounsel from casetext offers this service but it is $500 per month.  I would like to see this democratized and free to the public and lawyers to help increase access to justice making legal service more affordable.  I am currently running your webUI (fork from oobabooga) with a Vicuna 13b 4bit model and it works great, but lacks any real legal knowledge.  I also use GPT-4 and have tried webChatGPT and that also doesn't seem to be able to access any real library of good law.  What I would like would be to have an LLM with lanchain and serp to access google scholar.  This would essentially free up legal analysis to premit people to have better access to legal services and person legal research.  I expect this will happen one way or another but I would like to see it open source.
